variables.tf

```diff
@@ -140,7 +140,13 @@ variable "connection_draining_timeout_sec" {
 }
 
 variable "create_backend_firewall" {
-  description = "Controls if firewall rules for the backends will be created or not. Health-check firewall rules are always created."
+  description = "Controls if firewall rules for the backends will be created or not. Health-check firewall rules are controlled separately."
+  default     = true
+  type        = bool
+}
+
+variable "create_healtcheck_firewall" {
```

Comment:
```suggestion
variable "create_health_check_firewall" {
```

Reply:
Thanks for the answer. I did that, but all of them are skipped. I don't see any changes to the PR afterwards:
```
Generating markdown docs with terraform-docs
Working in . ...
Success!
Skipping ./examples/minimal because README.md does not exist.
Working in ./examples/simple ...
Success!
Skipping ./test/fixtures/minimal because README.md does not exist.
Skipping ./test/setup because README.md does not exist.
```

Reply:
Isn't it because this module doesn't really have a table with variables in readme?

Reply:
You're right, looks like you're fine.

Reply:
@morgante so you are good to give it 👍 ?